src/main/java/command/ExitCommand.java

```diff
@@ -0,0 +1,24 @@
+package command;
+
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+import duke.exception.DukeException;
+
+public class ExitCommand extends Command {
+    /**
+     * An override method to check whether the Command is an ExitCommand.
+     *
+     * @return Always return true indicating this is an ExitCommand.
+     */
+    @Override
+    public boolean isExit() {
+        return true;
+    }
+
```

Comment:
Maybe you can consider adding a Javadoc comment describing the execute() method here as well? This is to be consistent with your commenting for all the other execute() methods